kythe/cxx/common/status_or.h

```diff
@@ -58,34 +58,58 @@ class ABSL_MUST_USE_RESULT StatusOr final {
   template <typename U>
   StatusOr& operator=(StatusOr<U>&& other);
 
+  /// \brief Return this->status().ok()
   ABSL_MUST_USE_RESULT bool ok() const { return this->status_.ok(); }
 
-#if defined(__clang__) && __clang_major__ == 3 && __clang_minor__ == 5
-  // clang 3.5 will choose the incorrect overload if the ref-qualifiers
-  // are present.
-  // TODO: either bump the minimum version of clang we require or find a
-  // reasonable fix for this.
-  const Status& status() const { return this->status_; }
-  Status status() { return this->status_; }
-#else
+  /// \brief Return the contained status.
   const Status& status() const& { return this->status_; }
   Status status() && { return std::move(this->status_); }
-#endif
 
+  /// \brief Return this->ok()
```

Comment:
What are you intending to convey here?

Reply:
That the explicit operator bool is equivalent to the `ok()` member function.

Reply:
That might be a better way to put it than copying the implementation.